src/designer/core.js

```diff
@@ -30,12 +31,47 @@ exports.prototype = async function (blueprintInfo) {
     const userId = blueprint.user_id;
     const fileName = body.projectId;
 
+    let aiservicesCount = 0;
+    if (blueprint.metadata && blueprint.metadata.nodes) {
+        const { nodes } = blueprint.metadata;
+        Object.keys(nodes).forEach(key => {
+            if (key.startsWith("Service")) {
+                const service = nodes[key];
+                if (
+                    service?.data &&
+                    service?.data?.dbmlData &&
+                    service?.data?.description
+                ) {
+                    aiservicesCount++;
+                }
+            }
+        });
+    }
```

Comment:
Updated this to 
const dbmlCount = Object.values(services).filter(service => service.dbmlData).length;